lighthouse-core/lib/i18n/i18n.js

```diff
@@ -84,6 +84,10 @@ const UIStrings = {
   columnOverBudget: 'Over Budget',
   /** Label for a column in a data table; entries will be a representation of a DOM element. */
   columnElement: 'Element',
+  /** Label for a column in a data table; entries will be the number of milliseconds since the page started loading. */
+  columnStartTime: 'Start Time',
```

Comment:
technically not needed here but I say keep it b/c it will be needed here in your followup